src/helpers.test.js

```diff
@@ -405,3 +406,50 @@ describe('runLinter()', () => {
     expect(atom.commands.dispatch).not.toHaveBeenCalled();
   });
 });
+
+describe('getDebugInfo()', () => {
+  test('returns versions of prettier-atom and some of its direct dependencies', () => {
+    const atomVersion = '4.44.44';
+    const mockGetVersion = jest.fn(() => atomVersion);
+    const mockGet = jest.fn();
+    atom = {
+      getVersion: mockGetVersion,
+      config: { get: mockGet },
+    };
+
+    const minVersionLength = '0.0.0'.length;
+    const info = getDebugInfo();
+
+    expect(info.atomVersion).toBe(atomVersion);
+    expect(typeof info.prettierVersion).toBe('string');
+    expect(typeof info.prettierAtomVersion).toBe('string');
+    expect(typeof info.prettierESLintVersion).toBe('string');
+    expect(info.prettierVersion.length).toBeGreaterThanOrEqual(minVersionLength);
+    expect(info.prettierAtomVersion.length).toBeGreaterThanOrEqual(minVersionLength);
+    expect(info.prettierESLintVersion.length).toBeGreaterThanOrEqual(minVersionLength);
+  });
+
+  test('returns the configuration for prettier-atom', () => {
+    const expectedConfig = {
+      prettierOptions: {
+        printWidth: 120,
+        singleQuote: true,
+        semi: false,
+        parser: 'flow',
+      },
+      useEslint: false,
+      formatOnSaveOptions: {
+        enabled: false,
+      },
+    };
+    const mockGet = jest.fn(() => expectedConfig);
+    atom = {
+      getVersion: jest.fn(),
+      config: { get: mockGet },
+    };
+
+    const info = getDebugInfo();
+
+    expect(info.prettierAtomConfig).toMatchObject(expectedConfig);
+  });
```

Comment:
Can we combine this test in the one above? Also, I think we can use a simple string or something to make sure `prettierAtomConfig` is getting set to the right value and that would make it a bit shorter:

```js
const expectedConfig = 'atom prettier config';

// ... 

expect(info.prettierAtomConfig).toEqual(expectedConfig);
```